internal/flags/activate.go

```diff
@@ -21,7 +21,7 @@ func (f *Flags) handleActivateCommand() int {
 		return nil
 	})
 	// for local activation in ACM mode need a few more items
-	f.amtActivateCommand.StringVar(&f.configContent, "config", "", "specify a config file ")
+	f.amtActivateCommand.StringVar(&f.configContent, "configFile", "", "specify a config file ")
```

Comment:
I would prefer to leave the flag as config, so in the future, it would accept either string or file with our ongoing tech debt task.